tools/bazelify_tests/grpc_build_artifact_task.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/bash
+# Copyright 2023 The gRPC Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -e
+
+ARCHIVE_WITH_SUBMODULES="$1"
+BUILD_SCRIPT="$2"
+EXIT_CODE_FILE="$3"
+SCRIPT_LOG_FILE="$4"
+ARTIFACTS_OUT_FILE="$5"
+shift 5
+
+# Extract grpc repo archive
+tar -xopf ${ARCHIVE_WITH_SUBMODULES}
+cd grpc
+
+# Extract all input archives with artifacts into input_artifacts directory
```

Comment:
nit: this is copied into another script, and could be refactored into a reusable function.

Reply:
added a TODO.